lib/prompts/createFunctionPrompt.ts

```diff
@@ -2,8 +2,6 @@
 const { promptUser } = require('./promptUtils');
 const { i18n } = require('../lang');
 
-const { STRING_WITH_NO_SPACES_REGEX } = require('../regex');
```

Comment:
No need to use a regex to check for string spaces when we can do it easily with `indexOf()`